src/components/Player.ts

```diff
@@ -28,13 +28,13 @@ export default class Player extends Phaser.Sprite {
         this.tankrGame = tankrGame;
         this.health = 100;
         this.game.add.existing(this);
-        this.turret = this.game.add.sprite(0, 0, Images.ImgTanksTankBlueBarrel1.getName());
+        this.turret = this.game.add.sprite(0, 0, Images.ImgTanksTankBlueBarrel2Outline.getName());
         // turret rotates from middle of bottom, so set that as anchor
-        this.turret.anchor.setTo(0.5, 1);
+        this.turret.anchor.setTo(0.5, 0);
         this.anchor.setTo(0.5, 0.5);
         this.game.camera.follow(this);
         this.game.physics.enable(this, Phaser.Physics.ARCADE);
-
+        console.log(this.body);
```

Comment:
Please remove the `console.log`

Reply:
done, thank you! sorry, forgot about the log :(